### ISSUE TITLE
404 Page

### ISSUE THREAD

**Issue by abelhancock** [liferay-design/liferay.design#795]
**Is your feature request related to a problem? Please describe.**
Our 404 page is just a black screen (with black text you can't really see). This isn't helpful:( 

**Describe the solution you'd like**
Let's create a (legible) 404 page that includes the site nav/footer and helpful information/quick links.

**Describe alternatives you've considered**
Minimum improvement would be to make the existing text white and add a link to the home page:)

**Additional context**
![image](https://user-images.githubusercontent.com/31840274/93792061-e1a72200-fbe9-11ea-9e08-a1fb45887c97.png)
